addons/website/static/src/snippets/s_dynamic_snippet_carousel/000.xml

```diff
@@ -1,36 +1,60 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
+    <t t-name="website.s_dynamic_snippet.carousel.arrows">
+        <a t-attf-href="##{uniqueId}" class="carousel-control-prev" data-bs-slide="prev" role="button" aria-label="Previous" title="Previous">
+            <span class="fa fa-chevron-circle-left fa-2x"/>
+            <span class="visually-hidden">Previous</span>
+        </a>
+        <a t-attf-href="##{uniqueId}" class="carousel-control-next" data-bs-slide="next" role="button" aria-label="Next" title="Next">
+            <span class="fa fa-chevron-circle-right fa-2x"/>
+            <span class="visually-hidden">Next</span>
+        </a>
+    </t>
```

Comment:
I've tried the following multiple times:
- Create a DB in master with website_sale
- Drop a product snippet, select a template customize it
- Install my branch and run odoo with -u website,website_sale

The new templates are available, each card will have a new design, but the customization will remain (number of cards displayed on the screen)
The user can select a new template and it will apply the default options. They can also select the same template to have the default options re-applied.

Only thing that might not work is custom templates.
TL;DR default templates should work fine, unsure about custom templates, though I think they should work fine as well.

Reply:
:+1: